front/issue.php

```diff
@@ -54,17 +54,7 @@
    PluginFormcreatorCommon::showMiniDashboard();
 }
 
-//backup session value
-$save_session_fold_search = $_SESSION['glpifold_search'];
-//hide search if need
-if (PluginFormcreatorEntityconfig::getUsedConfig('is_search_issue_visible', Session::getActiveEntity()) == PluginFormcreatorEntityconfig::CONFIG_SEARCH_ISSUE_HIDDEN) {
-   $_SESSION['glpifold_search'] = true;
-}
-
-Search::show('PluginFormcreatorIssue');
-
-//restore session value
-$_SESSION['glpifold_search'] = $save_session_fold_search;
+PluginFormcreatorCommon::showGenericSearchIssueIfNeeded();
```

Comment:
I would avoid such name for a method. 

```suggestion
PluginFormcreatorCommon::showGenericSearchIssue();
```